docs/root/configuration/overview/overview.rst

```diff
@@ -21,14 +21,34 @@ support:
   when needed. The v2 APIs still maintain a baseline eventual consistency model.
 
 See the :ref:`xDS protocol description <xds_protocol>` for
-further details on aspects of v2 message exchange between Envoy and the management server.
+further details on aspects of xDS message exchange between Envoy and the management server.
 
-.. _config_overview_v2_bootstrap:
+Versioning
```

Comment:
Can you say something brief in here about how Envoy internally always uses the latest API version and thus will transparently receive v2, convert to v3, or convert to v3, without any config changes? Mainly I think operators need to understand that they can do management server upgrades for resources without requiring any Envoy config changes, which I think is super important to understand. I might even call this out in a sphinx note. (I think there is going to be a lot of confusion around the transport vs. resource and people are going to think they need to configure the V3 transport to get v3 resources)

Reply:
Sure, but operators probably will want to modify their `ApiConfigSource` (as per the examples) to express the transport or resource API versions. Added a note.